AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.ts

```diff
@@ -71,15 +79,28 @@ export class AllocateHearingsComponent implements OnInit {
     searchForHearings(keepExistingMessage: boolean = false) {
         const retrieveDate = (date: any): Date => (date === null || date === '' ? null : new Date(date));
 
-        const fromDate = retrieveDate(this.form.value.fromDate);
-        const toDate = retrieveDate(this.form.value.toDate);
+        let fromDateValue = retrieveDate(this.form.value.fromDate);
+        if (fromDateValue === null) {
+            fromDateValue = new Date();
+            this.form.patchValue({
```

Comment:
I don't understand why the form is being injected with date values when a user has not entered one themselves. My suggestion would be to either:

1. Modify the `retrieveDate` function to return `this.todayDate` instead of a null and leave the form as the user entered it
2. Initialise the `fromDate` and `toDate` properties in the constructor so the user has a clear understanding what dates are being set from the beginning

Reply:
The form is being injected with date values when a user has not entered one themselves so the user has an understanding of what is applied to retrieve the list. This has been agreed with Lorenzo and Lucy.
Need to ask to Lorenzo and Lucy if they are happy for the user to see populated the input boxes as default to today date for fromDate and 1 year date for toDate

Reply:
What's happen if the user wants only one day to be filtered? With your suggestion the user need to clear the toDate input field.

Reply:
This is working as design. Please check the ticket for agreed development. Happy to change it but it needs to be agreed by Lorenzo and Lucy.